lib/translate/common.h

```diff
@@ -3,5 +3,12 @@
 
 namespace translator {
 const char NVME_VENDOR_IDENTIFICATION[9] = "NVMe    ";
+
+static void (*debug_callback)(const char*);
+
+static void setDebugCallback(void (*callback)(const char*)) {
+  debug_callback = callback;
 }
+
+}  // namespace translator
```

Comment:
Probably better to wait for the version in https://github.com/googleinterns/scsi2nvme/pull/62 to be merged (or maybe split into a standalone PR for faster review/merge?) rather than having the same change in multiple PRs.

Reply:
that's a good idea. I will hold off on fixing optionals and look into creating a separate pr.